### PR TITLE
Clehner/wip fix tour

### PR DESCRIPTION
**Pull request by lehnerchristian**
the tour now considers the documents full height (including the scroll bar) and the scroll offset (because `mask.top` varies depending on the scroll offset)

### CODE REVIEW

**Comment by lehnerchristian** [comment]
@sgratzl , can you please have a look at this small PR? I need to test it in the deployed version, if it's ok

File: src/tour/TourManager.ts [hunk 1]
```diff
@@ -164,18 +164,28 @@ export default class TourManager {
...
   private setHighlight(mask: IBoundingBox) {
+    const fullAppHeight = this.backdrop.ownerDocument.body.scrollHeight; // full page height including scroll bars
+    const fullAppWidth = this.backdrop.ownerDocument.body.scrollWidth; // full page width including scroll bars
+
+    // set the new height of the backdrop
+    this.backdrop.style.height = `${fullAppHeight}px`;
+    this.backdrop.style.width = `${fullAppWidth}px`;
+
+    // also consider the current scroll offset inside the window
+    const scrollOffset = self.scrollY;
```
Comment (sgratzl) [inline src/tour/TourManager.ts]: `scrollY` is a property of which object?

File: src/tour/TourManager.ts [hunk 2]
```diff
@@ -164,18 +164,28 @@ export default class TourManager {
...
-      ${mask.left}px ${mask.top}px,
-      ${mask.left + mask.width}px ${mask.top}px,
-      ${mask.left + mask.width}px ${mask.top + mask.height}px,
-      ${mask.left}px ${mask.top + mask.height}px,
-      ${mask.left}px 100%,
-      100% 100%,
-      100% 0%
+      0% ${fullAppHeight}px,
+      ${mask.left}px ${fullAppHeight}px,
+      ${mask.left}px ${mask.top + scrollOffset}px,
```
Comment (sgratzl) [inline src/tour/TourManager.ts]: what happens when the user scrolls? or why is the scroll offset not considered when computing the mask?

File: src/tour/TourManager.ts (hunk 1 above)
Comment (lehnerchristian) [reply src/tour/TourManager.ts]: the window

**Pull request merged**